### ISSUE TITLE
Grayed out classification store fields/keys not included in GET query

### ISSUE THREAD

**Issue by MadeP** [pimcore/data-hub#342]
Hello,

grayed out classification store fields/keys (e.g. for variants) are not included in a datahub GET query?
Normal fields are exported even if they are grayed out and inherited from the main article. It would be good if this is the case for classification stores as well

Thanks in advance

**Comment by weisswurstkanone** [comment]
@MadeP you are talking about inherited fields ? Can this be reproduced on demo.pimcore.fun? Please provide a request sample.

**Comment by MadeP** [comment]
Yes, exactly. Unfortunately I don't know the access data for demo.pimcore.fun.However, I tried to reproduce it on https://demo.pimcore.com. strangely, the fields are not grayed out here, but still taken from the parent article. So they are also shown in the datahub get query. Was there a setting changed for this? Was this changed with an update?

Pimcore Demo:
![pimcore-demo-inherited-classificationstore](https://user-images.githubusercontent.com/66425228/120312715-5e41e280-c2d9-11eb-9d9a-8e0a904b239e.PNG)

My Pimcore:
![mypimcore-inherited-classificationstore](https://user-images.githubusercontent.com/66425228/120312756-6863e100-c2d9-11eb-8a7f-8dfbd739f79c.PNG)

**Comment by weisswurstkanone** [comment]
admin / demo 
which is on 10.0 right now. could you give it a try and provide feedback?

**Comment by MadeP** [comment]
I have tested it. Same result as described above. Btw the many-to-one relations are also not grayed out but are inherited from the parent article.

**Comment by weisswurstkanone** [comment]
Can you quickly provide a sample query which I can reproduce on demo.pimcore.fun? (30 mins left until it gets reset the next time)

**Comment by MadeP** [comment]
As already mentioned it works in the demo, because here the fields are not grayed out.

Using datahub "products" configuration:
```

{
  getCarListing(filter: "{\"$and\":[{\"o_type\":{\"$not\":\"variant\"}},{\"o_modificationDate\":{\"$gt\":1619092470}},{\"o_published\":\"1\"}]}") {
    edges {
      node {
        name
        test {
          features {
            ... on csFeatureSelect {
              group: name
              option: selection
            }
          }
        }
        children(objectTypes: ["variant"]) {
          ... on object_Car {
            name
            test {
              features {
                ... on csFeatureSelect {
                  group: name
                  option: selection
                }
              }
            }
          }
        }
      }
    }
  }
}
```

Is this possibly due to the parent php class? I don't use one in my pimcore environment

**Comment by weisswurstkanone** [comment]
No. And works as expected on demo.pimcore.fun for object 73. There is a value in the response because there is a value in the object. If you remove it then you will get null in the result because the parent is a folder so there cannot even be inheritance.
As already said we need a case on the demo which can be reproduced on our side.

**Comment by MadeP** [comment]
Can be closed. Problem apparently occurred only in Pimcore 6.8.x. After update, the classification stores for variants are handled differently and thus also interpreted correctly by datahub. Thanks for your efforts.

**Comment by weisswurstkanone** [comment]
Thank you!
